dagster_utils/resources/beam/k8s_beam_runner.py

```diff
@@ -49,10 +49,9 @@ def run(
         self,
         run_arg_dict: dict[str, Any],
         target_class: str,
-        scala_project: str
+        scala_project: str,
+        job_name: Optional[str] = None
```

Comment:
since this is an Optional, this wouldn't be a breaking change for existing projects using beamrunner, right?

Reply:
@quazi-broad yup, that's correct.